Policies/github-terms-of-service.md

```diff
@@ -71,8 +71,14 @@ The following information is required:
 Other information requested (which could include your real name) is optional.
 
 #### 3. Account Requirements
-- You must be a human to create an Account. Accounts registered by "bots" or other automated methods are not permitted. We do permit machine accounts. When you need to deal with, or know about, machine accounts, make sure you have read the following separately bullet-pointed text (otherwise you can skip it):  
-    * A machine account is an Account set up by an individual human who accepts the Terms on behalf of the Account, provides a valid email address, and is responsible for its actions. A machine account is used exclusively for performing automated tasks. Multiple users may direct the actions of a machine account, but the owner of the Account is ultimately responsible for the machine's actions. You may maintain no more than one free machine account in addition to your free User Account.
+- You must be a human to create an Account. Accounts registered by "bots" or other automated methods are not permitted. We do permit machine accounts. When you need to deal with, or know about machine accounts, make sure you have read the following collapsible text (otherwise you can skip it):  
+<details>
+    <summary>Click to toggle showing of collapsible text.</summary>
```

Comment:
No need to say this. Just put a summary there. You can make the block expanded by default by using `open="true"`. If a user doesn't need its content, he can just collapse it.

Reply:
Hello @KOLANICH,

The reason I felt it necessary to put such a summary (which is not a proper summary really), is for legal reasons.

We don't want legal claims against GitHub on the grounds that it wasn't clear how to read the collapsible text. Setting it to expanded by default would help, but then what if their browser software is faulty and instead renders it closed by default?

What are your thoughts about this?